app/assets/javascripts/copo/copo.edit-checkin.es6

```diff
@@ -87,24 +144,29 @@ window.COPO.editCheckin = {
 
   updateCheckin(response) {
     // tries to find the checkin in gon and update it with the response
-    checkin = _.find(gon.checkins, _.matchesProperty('id',response.id));
-    checkin.lat = response.lat;
-    checkin.lng = response.lng;
-    checkin.edited = response.edited;
+    checkin = _.find(gon.checkins, _.matchesProperty('id', response.checkin.id));
+    checkin.lat = response.checkin.lat;
+    checkin.lng = response.checkin.lng;
+    checkin.edited = response.checkin.edited;
     checkin.lastEdited = true;
-    checkin.address = response.address;
-    checkin.fogged_city = response.fogged_city;
```

Comment:
is there a way we can pass more options to the serializer so it sometimes does return attributes such as fogged_city? Fairly important that we have this functionality one way or the other.